generators/server/templates/pom.xml.ejs

```diff
@@ -108,6 +108,8 @@
         <log4j2-mock.version>0.0.1</log4j2-mock.version>
 <%_ } _%>
         <mapstruct.version>1.3.0.Final</mapstruct.version>
+        <!-- The hibernate version to be used to override default version of liquibase maven plugin -->
```

Comment:
This should be inside a conditional block : `<%_ if (databaseType === 'sql') { _%>`  because liquibase and hibernate are only used for sql databases.

Reply:
Thanks for reviewing @murdos